pkg/harvester/l10n/en-us.yaml

```
@@ -851,6 +851,8 @@ harvester:
       tip: IP Range should be IPv4 CIDR format and 4 times the number of your cluster nodes.
     vmForceDeletionPolicy:
       period: Period
+    httpProxy:
+      warning: "WARNING: <br/> If you configure httpProxy and httpsProxy, you must also put Harvester node's CIDR into noProxy, otherwise the Harvester cluster will be broken."
```

```suggestion
      warning: "WARNING: <br/> If you configure "http-proxy" and "https-proxy", you must also put the Harvester node's CIDR into "no-proxy", otherwise the Harvester cluster will be broken."
```

I think the prompt message should be named in a way that is consistent with the input box's label